### PR TITLE
remove extranous swidtag

### LINKED ISSUE #19250: extraneous swidtag
Currently, openliberty includes 2 .swidtags:
ibm.com_Open_Liberty-21.0.0.swidtag
openliberty.io_OpenLiberty.swidtag

The openliberty.io_OpenLiberty.swidtag should no longer be included.

### PR DESCRIPTION
**Pull request by bensonlatibm**
remove extraneous .swidtag (openliberty.io_OpenLiberty.swidtag)

### CODE REVIEW

**Comment by bensonlatibm** [comment]
#build

File: dev/com.ibm.ws.install.packaging_fat/fat/src/com/ibm/ws/install/packaging/fat/CheckTags.java [hunk 1]
```diff
@@ -134,10 +134,10 @@ public void testSwidtagsExist() throws Exception {
             unexpected_file_found = true;
         }
 
-        testsPassed = ol_swidtag_exists && IBM_ol_swidtag_exists && !unexpected_file_found;
+        testsPassed = !ol_swidtag_exists && IBM_ol_swidtag_exists && !unexpected_file_found;
```
Comment (jjiwooLim) [inline dev/com.ibm.ws.install.packaging_fat/fat/src/com/ibm/ws/install/packaging/fat/CheckTags.java:137]: Are we keeping `tag1` to make sure it doesn't exist? Then I think we should update the comments on line 106 and 119.

**Comment by bensonlatibm** [comment]
#build

**Review by jjiwooLim — APPROVED** [review]
LGTM!

**Comment by bensonlatibm** [comment]
#run-libby-bot

**Comment by bensonlatibm** [comment]
tested via pb.

**Pull request merged**